dm/pkg/schema/tracker.go

```diff
@@ -317,7 +352,10 @@ func (tr *Tracker) Reset() error {
 func (tr *Tracker) Close() error {
 	tr.se.Close()
 	tr.dom.Close()
-	return tr.store.Close()
+	if err := tr.store.Close(); err != nil {
+		return err
+	}
+	return os.RemoveAll(tr.storePath)
```

Comment:
i am not sure if this will affects other tracker in other running task in same worker

Reply:
storePath looks like `./schema-tracker11231231233`, `TempDir` should make sure each call returns unique dir